test/functional/sqlsrv/TC52_StreamSend.phpt

```diff
@@ -101,35 +101,58 @@ function SendStream($minType, $maxType, $atExec)
             fclose($fname1);
             $fname2 = fopen($fileName, "r");
 
-            $sqlType = GetSqlType($k);
-            $phpDriverType = GetDriverType($k, strlen($data));
-            $dataType = "[c1] int, [c2] $sqlType";
-            $dataOptions = array($k, array($fname2, SQLSRV_PARAM_IN, null, $phpDriverType));
-            TraceData($sqlType, $data);
-            CreateQueryTable($conn1, $tableName, $dataType, "c1, c2", "?, ?", $dataOptions, $atExec);
-            CheckData($conn1, $tableName, 2, $data);
+            $sqlType = getSqlType($k);
+            $phpDriverType = getDriverType($k, strlen($data));
+            traceData($sqlType, $data);
+
+            // create table
+            $columns = array(new AE\ColumnMeta('int', 'c1'),
+                             new AE\ColumnMeta($sqlType, 'c2'));
+            AE\createTable($conn1, $tableName, $columns);
+
+            // insert data
+            $params = array($k, array($fname2, SQLSRV_PARAM_IN, null, $phpDriverType));
+            insertQueryTable($conn1, $tableName, $params, $atExec);
+            checkData($conn1, $tableName, 2, $data);
 
             fclose($fname2);
         }
     }
-
-
     dropTable($conn1, $tableName);
+    unlink($fileName);
 
     sqlsrv_close($conn1);
 
     endTest($testName);
 }
 
-function CreateQueryTable($conn, $table, $dataType, $dataCols, $dataValues, $dataOptions, $execMode)
+function insertQueryTable($conn, $tableName, $params, $execMode)
```

Comment:
can you not use insertRow for this function?

Reply:
no because it's specific to this test, in which they set the option `SendStreamParamsAtExec` on and off for testing